firebase-firestore/src/main/java/com/google/firebase/firestore/remote/WatchChangeAggregator.java

```diff
@@ -196,17 +197,69 @@ public void handleExistenceFilter(ExistenceFilterWatchChange watchChange) {
               expectedCount == 1, "Single document existence filter with count: %d", expectedCount);
         }
       } else {
-        long currentSize = getCurrentDocumentCountForTarget(targetId);
+        int currentSize = getCurrentDocumentCountForTarget(targetId);
```

Comment:
count is an int in ExistenceFilter (both in proto and the class): 
`int32 count = 2;`

Change the long to int here to match.